skymp5-server/cpp/server_guest_lib/ActionListener.cpp

```diff
@@ -492,6 +492,7 @@ void ActionListener::OnChangeValues(const RawMessageData& rawMsgData,
     s += Networking::MinPacketId;
     s += nlohmann::json{
       { "t", MsgType::ChangeValues },
+      { "idx", actor->GetIdx() },
```

Comment:
Sorry? What is this change? This is out of the task's scope and even out of TS scope.